src/std/main.ab

```diff
@@ -266,3 +272,15 @@ pub fun shell_var_get(name: Text): Text {
 pub fun shell_unset(name: Text): Null {
     $unset {name}$?
 }
+
+pub fun btoa(text: Text): Text {
+    return unsafe $echo "{text}" | base64$
+}
+
+pub fun atob(encoded: Text): Text {
+    return unsafe $echo "{encoded}" | base64 --decode$
+}
+
+pub fun randint(min: Num, max: Num): Num {
+    return unsafe $shuf -i "{min}-{max}" -n 1$
```

Comment:
firstly you can correctly seed RANDOM variable by add currrent timestamp to process id
``unsafe $RANDOM=\$((\$(date +%s)+\$\$))$``
secondly return random number in range
``return unsafe $echo \$((\$((\$RANDOM%({max}-{min}+1)))+{min}))$ as Num``

this is not dependent from shuf command